src/components/MultiHopTrade/components/TradeQuotes/TradeQuote.tsx

```diff
@@ -1,15 +1,13 @@
 import { Flex, Skeleton, SkeletonCircle, Stack, Tag, useColorModeValue } from '@chakra-ui/react'
-import type { Result } from '@sniptt/monads/build'
 import { useCallback, useMemo } from 'react'
 import { FaGasPump } from 'react-icons/fa'
 import { useTranslate } from 'react-polyglot'
 import { Amount } from 'components/Amount/Amount'
 import { RawText } from 'components/Text'
 import { useIsTradingActive } from 'components/Trade/hooks/useIsTradingActive'
 import { bnOrZero } from 'lib/bignumber/bignumber'
-import type { SwapErrorRight } from 'lib/swapper/api'
+import type { SwapErrorRight, TradeQuote as TradeQuoteType } from 'lib/swapper/api'
```

Comment:
this might appear when you rebase, but in case its not
```suggestion
import type { SwapErrorRight, TradeQuote2 as TradeQuoteType } from 'lib/swapper/api'
```